code/modules/surgery/transorbital_lobotomy.dm

```diff
@@ -42,22 +42,22 @@
 	return ..()
 
 /datum/surgery_step/tlobotomize/failure(mob/user, mob/living/carbon/target, target_zone, obj/item/tool, datum/surgery/surgery)
-	var/obj/item/organ/brain/B = target.getorganslot(ORGAN_SLOT_BRAIN)
-	if(B)
-		display_results(user, target, "<span class='warning'>You remove the wrong part, causing more damage!</span>",
-			"<span class='notice'>[user] successfully lobotomizes [target]!</span>",
-			"<span class='notice'>[user] completes the surgery on [target]'s brain.</span>")
-		B.applyOrganDamage(80)
-		switch(rand(1,3))
-			if(1)
-				target.gain_trauma_type(BRAIN_TRAUMA_MILD, TRAUMA_RESILIENCE_MAGIC)
-			if(2)
-				if(HAS_TRAIT(target, TRAIT_SPECIAL_TRAUMA_BOOST) && prob(50))
-					target.gain_trauma_type(BRAIN_TRAUMA_SPECIAL, TRAUMA_RESILIENCE_MAGIC)
-				else
-					target.gain_trauma_type(BRAIN_TRAUMA_SEVERE, TRAUMA_RESILIENCE_MAGIC)
-			if(3)
-				target.gain_trauma_type(BRAIN_TRAUMA_SPECIAL, TRAUMA_RESILIENCE_MAGIC)
-	else
-		user.visible_message("<span class='warning'>[user] suddenly notices that the brain [user.p_they()] [user.p_were()] working on is not there anymore.</span>", "<span class='warning'>You suddenly notice that the brain you were working on is not there anymore.</span>")
-	return FALSE
+    var/obj/item/organ/brain/Brian = target.getorganslot(ORGAN_SLOT_BRAIN)
+    if(!Brain)
```

Comment:
This is `Brian` first and `Brain` after. Subtle difference.
Use snake_case for variables if you can, instead of PascalCase